tailwind.config.js

```diff
@@ -1,12 +1,13 @@
 module.exports = {
   content: {
-    relative: true,
-    files: [
-      './resources/**/*.blade.php',
-      './resources/**/*.js',
-      './resources/**/*.vue',
-    ],
-  },
+      relative: true,
+      files: [
+        './resources/**/*.blade.php', 
+        './resources/**/*.js', 
+        './resources/**/*.vue',
+        './vendor/laravel/framework/src/Illuminate/Pagination/resources/views/*.blade.php',
```

Comment:
why? 
u can create that in our resources.

Reply:
deleted this